src/components/laser-controls.js

```diff
@@ -16,21 +16,36 @@ registerComponent('laser-controls', {
     el.setAttribute('gearvr-controls', {hand: data.hand});
     el.setAttribute('oculus-touch-controls', {hand: data.hand});
     el.setAttribute('vive-controls', {hand: data.hand});
+    el.setAttribute('windows-motion-controls', {hand: data.hand});
 
-    // Wait for controller to connect before
-    el.addEventListener('controllerconnected', function (evt) {
+    // Wait for controller to connect, or have a valid pointing pose, before creating ray
+    el.addEventListener('controllerconnected', createRay);
+    el.addEventListener('controllermodelready', createRay);
+
+    function createRay (evt) {
       var controllerConfig = config[evt.detail.name];
 
       if (!controllerConfig) { return; }
 
-      el.setAttribute('raycaster', utils.extend({
+      // Show the line unless a particular config opts to hide it, until a controllermodelready
+      // event comes through.
+      var raycasterConfig = utils.extend({
         showLine: true
-      }, controllerConfig.raycaster || {}));
+      }, controllerConfig.raycaster || {});
+
+      // The controllermodelready event contains a rayOrigin that takes into account
+      // offsets specific to the loaded model.
+      if (evt.detail.rayOrigin) {
+        raycasterConfig = utils.extend(raycasterConfig, evt.detail.rayOrigin);
```

Comment:
is it clear to do? 
````javascript
raycasterConfig.reayOrigin = evt.detail.rayOrigin;
````

Reply:
you'd need to do something like..
```
raycasterConfig.origin = evt.detail.rayOrigin.origin
raycasterConfig.direction = evt.detail.rayOrigin.direction
```

but we expand that out if preferred?

Reply:
yeah, probably better to make it explicit to avoid surprises. nothing hidden 🙂